tests/modpow.rs

```diff
@@ -81,6 +81,7 @@ mod biguint {
         check_modpow::<u32>(0, 15, 11, 0);
         check_modpow::<u32>(3, 7, 11, 9);
         check_modpow::<u32>(5, 117, 19, 1);
+        check_modpow::<u32>(20, 1, 2, 0);
```

Comment:
Please add a similar test with an odd modulus too, so that corner case is not missed. (But I think that implementation is already OK.)

Reply:
Additionally, I added api surface test for small values below